src/doc/contrib/src/team.md

```diff
@@ -0,0 +1,156 @@
+# Cargo Team
+
+## Mission
+
+The Cargo Team is a group of volunteers that support the Rust community in developing and maintaining Cargo, the Rust package manager and build tool.
+The team is responsible for deciding how Cargo and its related libraries operate and evolve.
+The team has a shared responsibility with the [crates.io team] for the design and usage of Cargo's index format and its registry API as it relates to the [crates.io] service.
+
+The team is expected to keep Cargo in an operational state, to support Rust's 6-week release cycle, and to uphold the [Design Principles] of Cargo.
+
+[crates.io team]: https://www.rust-lang.org/governance/teams/crates-io
+[crates.io]: https://crates.io/
+[Design Principles]: design.md
+
+## Team membership
+
+The Cargo Team consists of team members with one serving as a team leader.
+The team leader is responsible for coordinating the team and providing a contact point with other teams.
+The leader is selected by a vote of the existing members.
+
+Membership is maintained in the [Rust team database].
+
+[Rust team database]: https://github.com/rust-lang/team/blob/master/teams/cargo.toml
+
+### Membership expectations
+
+Team members are expected to participate in voting on RFCs and major change proposals
+
+Team members are expected to regularly participate in at least some of the following membership-related activities.
+Members are not expected to participate in all of these activities, but exhibit some interest and involvement in the project that covers some of these activities.
+
+- Attending meetings
+- Reviewing contributions (auto-assignment is managed in [triagebot.toml])
+- Triaging and responding to issues
+- Mentoring new contributors
+- Shepherding major changes and RFCs
+- Coordinating interaction with other Rust groups and outside interests
+- Managing and updating the policies of the Cargo Team itself
+- Keeping up with maintenance of the Cargo codebase, ensuring it stays functional and that infrastructure and team processes continue to run smoothly
+
+Breaks and vacations are welcome and encouraged.
+If a member is no longer participating after a few months, they may be asked to step down.
+
+Members are required to always:
+
+- Represent the Rust project in a way that upholds the [Rust code of conduct][coc] to a high standard.
+- Represent the Cargo Team in a way that upholds the expectations of this charter, and be friendly, welcoming, and constructive with contributors and users.
+
+Members are given privileges, such as:
```

Comment:
One thing missing here is the permission of crates.io for crates maintained by the Cargo team. Who has the right to publish manually? Or should we make all things in automation even for a point release of cargo-the-library?

Reply:
If we adopt cargo-release, it would semi-automate the process. One of the more recent features is you can set a workspace-wide cargo-owner and it will be set when publishing new crates. I added this when I I realized there were newer clap crates without the team added to them.

Reply:
At this time I don't believe the infra team is willing to support tokens for auto-publishing from CI.  They are at least waiting on scoped tokens. 

I'm not sure how best to manage crates.io ownership. I think for now people can be added on an as-needed basis. I think we should preferably keep the ownership list small, perhaps eventually with no individual owners.  I'm not sure if we can fully automate it once scoped tokens are available (such as figuring out how to trigger releases, etc.).

Reply:
> At this time I don't believe the infra team is willing to support tokens for auto-publishing from CI. They are at least waiting on scoped tokens.

Experimental support for scoped tokens has shipped, so it's possible to do this now.